tests/unit/src/Internal/Service/PaymentProcessingServiceTest.php

```diff
@@ -107,11 +109,90 @@ public function test_process_payment_happy_path_without_mock_builder() {
 			->willReturn( $mock_initial_state );
 
 		$mock_initial_state->expects( $this->once() )
-			->method( 'process' )
+			->method( 'start_processing' )
 			->with( $this->isInstanceOf( PaymentRequest::class ) )
 			->willReturn( $mock_completed_state );
 
 		$result = $sut->process_payment( 1 );
 		$this->assertSame( $mock_completed_state, $result );
 	}
+
+	public function test_get_authentication_redirect_url_will_return_url_from_payment_intent() {
+		$sut = new PaymentProcessingService( $this->mock_state_factory, $this->mock_legacy_proxy );
```

Comment:
`$sut` is defined explicitly in the previous test because it tests `create_payment_context` (that's mocked normally). For this and all following tests, you can use `$this->sut` directly.

Reply:
Addressed here: https://github.com/Automattic/woocommerce-payments/pull/7471/commits/b35ba641977dfd7e93c02b85d336adae06f2dcf7